### DISCUSSION TITLE
[Question]  Unfindable newly created address / account on subscan

### DISCUSSION THREAD

**Discussion by Gregordy** [darwinia-network/darwinia#638] [Q&A] [answered]
I created an account but the [address](2oVrtRZAihPsEaM7mzRxyoTgpGTpBWgjhtJMTY5Xy8STr5zq) that was given to me afterwards is not findable on [subscan](https://darwinia.subscan.io/account/2oVrtRZAihPsEaM7mzRxyoTgpGTpBWgjhtJMTY5Xy8STr5zq).

Is there anything I missed (an indexing process ranning every X hour/day?) or is it an ongoing issue that is being worked out atm?

Thank you 😊

**Comment by aurexav** [comment, marked as answer]
At least own 0.1 RING. Otherwise your account won't be really created on the chain.

**Reply by Gregordy** [reply]
Oh yeah got it, that's what I thought for a moment.

I'm using [wormhole](https://wormhole.darwinia.network/crosschain/#ethereum) to transfer my RING from my ERC-20 wallet to the Darwinia one.

I'm on the right track right?

I can safely send my RING to this address even though it's not created yet right?

Thanks a lot for such a quick reply!

**Reply by aurexav** [reply]
Once the cross-chain transfer success. Your account will be funded then it will be created.
